gendoc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+DOC_DIR=godoc
+
+# Run a godoc server which we will scrape. Clobber the GOPATH to include
+# only our dependencies.
+GOPATH=$(pwd):$(pwd)/vendor godoc -http=localhost:6060 &
+DOC_PID=$!
+
+# Wait for the server to init
+sleep 1
```

Comment:
Might be nice to put a while loop here in case the godoc server takes longer to start?